doc/content/integrations/cloud-integrations/my-iot-open-tech/_index.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "my IoT open Tech"
+description: ""
+weight:
+---
+
+my IoT open Tech is an IoT platform, developed on top of ThingsBoard Community Edition (Open Source), with a user-centered design.
```

Comment:
can you link some kind of resource for ThingsBoard Community Edition? just so that the user that reads this doesn't have any inclarities about which platform exactly that is, and so that he doesn't have to google it up on his own

Reply:
Thanks @nejraselimovic 
I have included a link to ThingsBoard and accepted all your other suggestions.
Kind regards